rust/agama-lib/share/profile.schema.json

```diff
@@ -303,41 +303,98 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "boot": {
+          "$ref": "#/$defs/boot"
+        },
+        "drives": {
+          "title": "Drive devices",
+          "description": "Section describing drives (disks, BIOS RAIDs and multipath devices).",
+          "type": "array",
+          "items": {
+            "anyOf": [
+              {
+                "title": "Unpartitioned drive",
+                "description": "Drive without a partition table (e.g., directly formatted).",
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "search": {
+                    "description": "The search is limited to drives scope.",
+                    "$ref": "#/$defs/search"
+                  },
+                  "encryption": {
+                    "$ref": "#/$defs/encryption"
+                  },
+                  "filesystem": {
+                    "description": "The partition table (if any) is deleted.",
+                    "$ref": "#/$defs/filesystem"
+                  }
+                }
+              },
+              {
+                "title": "Partitioned drive",
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "search": {
+                    "description": "The search is limited to drives scope.",
+                    "$ref": "#/$defs/search"
+                  },
+                  "encryption": {
+                    "$ref": "#/$defs/encryption"
+                  },
+                  "ptableType": {
+                    "title": "Partition table type",
+                    "description": "The partition table is created only if all the current partitions are deleted.",
+                    "enum": ["gpt", "msdos", "dasd"]
+                  },
+                  "partitions": {
+                    "$ref": "#/$defs/partitions"
+                  }
+                }
+              }
+            ]
+          }
+        },
         "guided": {
-          "title": "Settings to execute a Guided Proposal",
+          "title": "Guided proposal settings",
+          "$comment": "Guided settings will be removed from this schema.",
```

Comment:
Not sure whether the new title is better than the previous. But, in any case, the `$comment` could be more clear.

Reply:
In fact, I would like to avoid the proposal word. The user of the config only has to know about configs, nothing about internals like proposal.

Reply:
Regarding the title, I am trying to use something short, and a description for longer explanations.